src/components/media/MediaCard.jsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import mediaStyles from '../../styles/isomer-cms/pages/Media.module.scss';
+import {fetchImageURL} from "../../utils";
+import {useQuery} from "react-query";
 
 const MediaCard = ({
   type, siteName, onClick, media, isSelected, canShowEditIcon,
-}) => (
+}) => {
+
+  const {data: imageURL, status} = useQuery(media.path, () => fetchImageURL(siteName, media.path), {
+    refetchOnWindowFocus: false,
+    staleTime: 1000 * 60 // 60 seconds
+  })
```

Comment:
Could we specify this query to only run on image type MediaCards? Currently we make a lot of failed calls to the backend for other types of media

Reply:
Added a `shouldLoad` parameter to the `fetchImageURL` method. Tried setting `enabled: type === 'images'` in the `useQuery` options but the MediaCards in Files somehow still trigger failed `fetchImageURL` calls after navigating from Files to Images.